pypeit/core/coadd.py

```diff
@@ -2575,15 +2628,15 @@ def ech_combspec(waves_arr_setup, fluxes_arr_setup, ivars_arr_setup, gpms_arr_se
     # Decide how much to smooth the spectra by if this number was not passed in
     nspec_good = []
     ngood = []
-    if sn_smooth_npix is None:
-        # Loop over setups
-        for wave, norder, nexp in zip(waves_arr_setup, norders, nexps):
-            # This is the effective good number of spectral pixels in the stack
-            nspec_good.append(np.sum(wave > 1.0))
-            ngood.append(norder*nexp)
-        nspec_eff = np.sum(nspec_good)/np.sum(ngood)
-        sn_smooth_npix = int(np.round(0.1 * nspec_eff))
-        msgs.info('Using a sn_smooth_pix={:d} to decide how to scale and weight your spectra'.format(sn_smooth_npix))
+    #if sn_smooth_npix is None:
```

Comment:
Can we remove this commented code? Or add in a note to keep it in (so it doesn't get unintentionally deleted)?

Reply:
would like to keep for now, in case we need to think about how to compute this for echelle again.